### ISSUE TITLE
New release that supports AWS SSO

### ISSUE THREAD

**Issue by oofnikj** [awslabs/amazon-ecr-credential-helper#290]
Since #229 was resolved several months ago, I'd like to request cutting a new release of this tool with the improvements merged since. It's not always convenient or possible to build from source.

**Comment by sergeylanzman** [comment]
@kzys @samuelkarp 
Please release new version of  `amazon-ecr-credential-helper`. 
Last release was on Feb 15... 
SSO support it's huge blocker for switch use AWS sso in our company.

**Comment by tomaszdudek7** [comment]
We need that desperately.

**Comment by rlangfordBV** [comment]
New release please :)

**Comment by whyman** [comment]
Pretty please? With a 🍒 on top?

**Comment by dancmeyers** [comment]
This would be amazing!

**Comment by jessebye** [comment]
The PR was merged in March. That makes it about 6 months now with no releases. Has this project been abandoned? Is it time to fork and publish our own version?

**Comment by kyledecot** [comment]
🤞 for a new release.

**Comment by jessebye** [comment]
If it helps anyone, I've forked this repo and created a new release that should support SSO:
https://github.com/jessebye/amazon-ecr-credential-helper/releases/tag/v0.6.0

**Comment by calebwhitt** [comment]
Thank you @jessebye!

Would be great to know if the original amazon-ecr-credential-helper has been abandoned too. Any updates?

**Comment by kzys** [comment]
Sorry for the very late reply. No, the helper hasn't been abandoned. We are going to have a new release to include unreleased changes.

That being said, we are internally discussing the model to support the helper more proactively. We've been slow to respond PRs and issues and we'd like to change that.

**Comment by calebwhitt** [comment]
Thanks for the update @kzys, much appreciated :)

**Comment by daniel-farrell-blockfi** [comment]
I was able to rebuild this and get it working.  If anyone wants the binary I ended up with for `darwin`, it's at https://github.com/daniel-farrell-blockfi/amazon-ecr-credential-helper/releases/tag/0.6.3

**Comment by icorbett** [comment]
Is there any reason not to tag a new release during the time that internal discussions are happening? This project seems to be quite stable even if its future is less certain. With SSO being increasingly common, this seems like an unnecessary stumbling block.

**Comment by jmriebold** [comment]
This is an issue for us as well, as our whole organization has just switched to SSO.

**Comment by jobimrobinsantos-drizly** [comment]
I'll just go ahead and add my voice to the chorus here. My org has switched to using SSO for user management, and while we await a new formal release we are using `go install`, which is not ideal.

For those that it might help:
```sh
GOPATH="$HOME/go" go install github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/docker-credential-ecr-login@main

# then make sure $HOME/go/bin is in your $PATH or create a symlink:
ln -s ~/go/bin/docker-credential-ecr-login SOMEWHERE_IN_YOUR_$PATH/docker-credential-ecr-login
```

**Comment by Klaitos** [comment]
We definitely need that release in my org too.

**Comment by lachlancooper** [comment]
[v0.6.0](https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/v0.6.0) has been released with SSO support 🎉

**Comment by raul-verdi** [comment]
perhaps this issue can be closed @oofnikj ?

**Comment by kzys** [comment]
Sorry for the very late reply. Yes. We've released v0.6.0. Closing.
